doc/nrf/ug_matter_overview_dfu.rst

```diff
@@ -0,0 +1,189 @@
+.. _ug_matter_overview_dfu:
+
+Matter OTA
+##########
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Matter Over-The-Air is a process that allows a Matter device in a Matter fabric to update its firmware.
```

Comment:
I am missing one  information here or there.. that Matter OTA is a process described by the Matter specification. Although we have a name "Matter" - this might not be obvious.